compsat/ls-ganap
# Feature/user jwt authentication

Reset db before running
docker volume rm api_pgdata

Test that it is able to:

1. generate a token 
2. refresh a token
3. verify a token
4. only allow posting events if the user is authenticated
